lumigator/jobs/inference/model_clients/huggingface_clients.py

```diff
@@ -51,6 +51,7 @@ def _initialize_model_and_tokenizer(self, config):
             tokenizer=self.tokenizer,
             revision=config.hf_pipeline.revision,
             device=config.hf_pipeline.device,
+            use_auth_token=config.api_key,
```

Comment:
```suggestion
            token=config.api_key,
```

`use_auth_token` appears to be deprecated:

https://github.com/huggingface/transformers/blob/6bc0fbcfa7acb6ac4937e7456a76c2f7975fefec/src/transformers/pipelines/__init__.py#L762